homeassistant/auth/__init__.py

```diff
@@ -446,6 +446,15 @@ async def async_remove_refresh_token(
         """Delete a refresh token."""
         await self._store.async_remove_refresh_token(refresh_token)
 
+    @callback
+    def async_register_revoke_token_callback(
+        self, refresh_token_id: str, callback: CALLBACK_TYPE
+    ) -> CALLBACK_TYPE:
+        """Register a callback to be called when the refresh token id is revoked."""
+        return self._store.async_register_revoke_token_callback(
```

Comment:
Auth store is for storing data. We should track these callbacks on `self` (`AuthManager`)

Reply:
Moved to AuthManager here 38fa6de